java/core/src/main/java/co/worklytics/psoxy/rules/salesforce/PrebuiltSanitizerRules.java

```diff
@@ -27,122 +24,168 @@ public class PrebuiltSanitizerRules {
                     .jsonPath("$..Email")
                     .jsonPath("$..Name")
                     .jsonPath("$..Username")
+                    .applyOnlyWhen("$..records[?(@.attributes.type == \"User\")]")
```

Comment:
why `..` not just `.`?  Haven't profiled it, I just assume `..` is pretty inefficient as it needs to recursively check the whole hierarchy for matching nodes, not just one level.